common/src/models/mlcommons_tiny_v01/vww/vww.cc

```diff
@@ -52,7 +52,7 @@ int32_t vww_classify() {
   static void name() {                                                 \
     puts(#name);                                                       \
     tflite_set_input(mlcommons_tiny_v01_vww_dataset[test_index].data); \
-    printf("  result-- score: %d\n", vww_classify());                  \
+    printf("  result-- score: %ld\n", vww_classify());                  \
```

Comment:
nit: doesn't really matter but consider removing one space to keep the alignment of `\`.
```suggestion
    printf("  result-- score: %ld\n", vww_classify());                 \
```